### DISCUSSION TITLE
Equipamento de Escritório - Ergonomia - LER - Saúde

### DISCUSSION THREAD

**Discussion by mjacobus** [DyegoCosta/trabalhando-remoto#187] [General]
Bom dia a todos!

Estou pensando em fazer um upgrade de mesa/cadeira devido a dores nos ombros e nas costas. Não tenho certeza, mas o provável motivo é a grande quantidade de horas gastas em frente ao computador. Minha mesa, cadeira, suporte para monitores não são de baixa qualidade, mas não são o que há de melhor em questão de ergonomia.

Estou cogitando comprar uma standup desk com motor de suspensão, mas não sei se vai valer o investimento. O equipamento é caro. Quando trabalhava no escritório eu tinha uma, mas raramente levantava. No entanto, devido as dores que sinto hoje tenho muita vontade de trabalhar de pé, pois sinto que as dores aliviam quando levanto.

- Alguém aqui utiliza standup desk pelos mesmo motivos (dores no corpo, LER, etc)? Teve algum benefício?
- E cadeiras, alguém tem alguma sugestão de bom custo benefício?
- Talvez outras dicas de como manter o escritório confortável pra quem passa a maior parte do dia programando.

Obrigado.

**Comment by copercini** [comment]
Eu tinha dores nos pulsos, então comprei um mouse pad ergonômico e resolveu o problema de uma mão kkkk 
depois tive que comprar um apoio de pulso para teclado e nunca mais senti dores nos pulsos

Para ilustrar:

![apoio](https://user-images.githubusercontent.com/21266776/83297984-035b4400-a1ca-11ea-8e40-b4b83b83bea4.jpg)

**Comment by rubemz** [comment]
@mjacobus eu tenho uma standup desk e comprei uma na [geniodesks](https://www.geniodesks.com.br/) uns dois anos atrás. O atendimento e a entrega foram muito bons. E tento trabalhar parte do dia em pé. As dores nas costas diminuiram.

> Talvez outras dicas de como manter o escritório confortável pra quem passa a maior parte do dia programando.

Eu uso um notebook num suporte e um monitor externo. O setup que ficou bom pra mim foi manter a tela do notebook e o monitor na altura dos meus olhos. Antes eu tinha dores no pescoço por ficar olhando pra baixo. Uso um teclado da Razer mecânico com suporte para os punhos, já que o notebook fica no suporte agora. 

 O suporte do teclado me ajuda não ter dores nos punhos também (o teclado é um Razer Blackwidow Tournament V2 Chroma e gosto muito dele)

**Comment by mjacobus** [comment]
Obrigado @copercini e @rubemz pelas dicas :)

**Comment by yuriploc** [comment]
> @mjacobus eu tenho uma standup desk e comprei uma na [geniodesks](https://www.geniodesks.com.br/) uns dois anos atrás. O atendimento e a entrega foram muito bons. E tendo trabalhar parte do dia em pé. As dores nas costas diminuiram.
> 
> > Talvez outras dicas de como manter o escritório confortável pra quem passa a maior parte do dia programando.
> 
> Eu uso um notebook num suporte e um monitor externo. O setup que ficou bom pra mim foi manter a tela do notebook e o monitor na altura dos meus olhos. Antes eu tinha dores no pescoço por ficar olhando pra baixo. Uso um teclado da Razer mecânico com suporte para os punhos, já que o notebook fica no suporte agora.
> 
> O suporte do teclado me ajuda não ter dores nos punhos também (o teclado é um Razer Blackwidow Tournament V2 Chroma e gosto muito dele)

Qual switch você usa? E o SO? Me recomendaram muito o Keychron, mas o suporte dele pra Linux é bem limitado.

**Comment by rubemz** [comment]
@yuriploc Switch laranja e uso o Linux (Ubuntu)

Tem como instalar um daemon e um programa com uma interface básica pra controlar as cores do teclado, etc. Não se compara com o suporte do windows, mas funciona ok no ubuntu.

Tem esse projeto chamado [OpenRazer](https://openrazer.github.io/) onde tu pode ver como instalar/configurar o teclado da Razer no linux.

**Comment by mjacobus** [comment]
Troquei o me teclado Microsoft Natural Ergonomic 4000 pelo Ergodox-ez. Parece que já aliviou um pouco as dores no ombro. Estou no aguardo da minha geniodesk. São 35 dias úteis para entrega :cry: 

Eu acho que a mesa vai ajudar muito. Pouco tempo depois de teu ter aberto este tópico, tive uma semana com dores terríveis na perna, na região em volta da virilha. Como se alguém tivesse puxado ela pra fora do socket ou se eu tivesse tentado fazer um espacate e tivesse rasgado alguma coisa lá dentro. Com medo de ir para o hospital fazer consulta na parte de trauma, devido a pandemia, acabei esperando a dor passar, e não passou de jeito nenhum. Dava vontade de gritar de dor, terrível levantar da cama ou fazer certos movimentos. Eu sentia o corpo pedino para ficar de pé, e eu literalmente usei a tábua de passar roupa como standing desk. Ficava variando posições a cada 20 ou 30 minutos. Parecia aliviar a dor.

Segundo o "diagnóstico" do farmacêutico, isto pode ter sido o rompimento de uma membrana ali pra dentro, no púbis. Ele já teve isto ao jogar futebol sem aquecer e disse que foram uns 6 meses até ficar bom.

Tomei anti-inflamatórios e parece ter resolvido. Melhorou muito. No final de semana a dor quis dar o ar da graça, mas parece que voltou ao normal.

Desconfio que a lesão tenha sido causada pelo meu costume de sentar em cima dos pés, ou de manter as pernas cruzadas daquele jeito que dizem que homem não consegue.

Aguardo ansiosamente a mesa.

**Comment by gabrielponto** [comment]
Em 2013 eu pesquisei bastante sobre cadeiras e comprei a Ergohuman. Nunca mais tive nenhuma dor nas costas desde então.

Eu troquei o pistão e as rodinhas esse ano, ela ainda está ótima. Recomendo demais.

Agora estou pensando também em trabalhar em pé, mas meu desejo maior seria uma esteira, pois assim eu passaria o dia caminhando :) - Já que estou muito indisposto em fazer exercícios diariamente.

Algumas pesquisas apontam que não há muitos benefícios em trabalhar por muito tempo em pé, comparado a ficar muito tempo sentado. E o que teria mais efeito é praticar atividades físicas. Eu já experimentei e tive bom resultado trabalhando em pé (no meu caso eu improvisei mesmo com livros para fazer o suporte).

**Comment by pedrohyvo** [comment]
Eu utilizo um suporte para o notebook muito bom e recomendo demais, é esse abaixo:
![image](https://user-images.githubusercontent.com/15241188/103598748-671cbc80-4ed1-11eb-967f-d2b2abddab3c.png)

Já o mouse/teclado via bluetooh porque eu coloco da forma que eu achar mais conveniente para trabalhar e ambos são da Logitech.

**Comment by mjacobus** [comment]
Eu acabei comprando uma standup desk e não me arrependo. Ainda passo a maior parte do tempo sentado, mas levanto algumas vezes no dia. Procuro ficar de pé durante reuniões também.

Também comprei um teclado, o Ergodox EZ. Este eu me arrependi amargamente por duas semanas, pois apanhei muito até conseguir configurar um layout que não matasse totalmente a minha produtividade e o teclado mais importação e taxas de CC me custaram cerca de 3.5k. Agora me acostumei com ele, e lógico, desaprendi a usar teclados normais, então tem tudo isto a se colocar na balança.

Sinto que estes dois investimentos me ajudaram bastante. Sinto menos dores. Também procuro fazer exercícios.

Sinto que minha cadeira passou a não ser mais tão confortável quanto ela era, e sinto desconforto na região lombar, como se a cadeira não me desse o suporte necessário. Acho que logo vou ter que investir em uma nova cadeira. O problema é que comprar cadeira sem experimentar é bem diferente que comprar uma mesa... vou avaliar as opções. Obrigado pela sugestão da Ergohuman, @gabrielponto .

Tenho também os suportes pra monitores, mas ainda não aprendi a não me inclinar e encostar o nariz neles :man_shrugging: 

![5352dd92-5379-468e-bbf8-21309e6a8702](https://user-images.githubusercontent.com/226834/103644405-13dc5580-4f35-11eb-9443-1ac95856d472.jpeg)

![8106051b-86bf-405c-b064-2bea1a5284e0](https://user-images.githubusercontent.com/226834/103644401-12ab2880-4f35-11eb-9627-09d06909257f.jpeg)

**Comment by DyegoCosta** [comment]
@mjacobus o que achou da mesa da geniodesks? 

Se ainda estiver procurando cadeira recomendo a Aeron da Herman Miller (estava na promoção até poucos dias atrás, eles colocam na promoção de tempos em tempos parece).

**Comment by mjacobus** [comment]
@DyegoCosta não me arrependi de ter comprado! Eu imagino que eu troque de posição umas 5 ou 6 vezes por dia. Ajuda bastante a se sentir confortável na frente do computador, e as dores no ombro diminuíram - mas não sei qual foi o fator dominante (teclado, mesa, exercícios...)

Vou dar uma conferida na cadeira. Obrigado pela dica!

**Comment by mjacobus** [comment]
@DyegoCosta, não é que tá barato mesmo?! 😄 

![Screen Shot 2021-03-11 at 11 07 38](https://user-images.githubusercontent.com/226834/110799573-0fe0f280-825a-11eb-826b-5f3ef68e7ea4.png)

Qual é o valor da promoção que tu encontrou?

**Comment by DyegoCosta** [comment]
> @DyegoCosta, não é que tá barato mesmo?! 😄
> 
> Qual é o valor da promoção que tu encontrou?

Vish hehe encontrei por 9k mas é o modelo Grafite, se você mudar a "cor" aí vai ver que a primeira é mais barata
